Configuration/Generator/python/SingleNuE10_pythia8_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+generator = cms.EDProducer("FlatRandomEGunProducer",
+    PGunParameters = cms.PSet(
+        PartID = cms.vint32(12),
+        MaxEta = cms.double(2.85),
+        MaxPhi = cms.double(3.14159265359),
+        MinEta = cms.double(-2.85),
+        MinE = cms.double(9.99),
+        MinPhi = cms.double(-3.14159265359), ## in radians
+
+        MaxE = cms.double(10.01)
+    ),
+    Verbosity = cms.untracked.int32(0), ## set to 1 (or greater)  for printouts
+
+    psethack = cms.string('single Nu E 10'),
+    AddAntiParticle = cms.bool(False),
+    firstRun = cms.untracked.uint32(1)
+)
```

Comment:
This file is duplicated to the existing one. We do not need to PR this config file. @siddhesh86